### PR TITLE
Different CTAs

### PR DESCRIPTION
**Pull request by adamgordonbell**
Here's an example of what the on page changes look like:

https://643de9041830553751f54257--ephemeral-narwhal-3ac3b2.netlify.app/blog/docker-networking/

Screen Shot 2023-04-17 at 8 51 48 PM

article ... 
article ... 
article ...
Screen Shot 2023-04-17 at 8 52 28 PM

### CODE REVIEW

File: blog/_posts/2021-08-24-jq-select.md [hunk 1]
```diff
@@ -375,8 +374,8 @@ If your `jq` query returns more than one element, they will be returned newline
 
 ~~~{.bash caption=">_"}
 $ echo '[{"a":"b"},{"a":"c"}]' | jq -r '.[].a'
-  "b"
-  "c"
+  b
```
Comment (adamgordonbell) [inline blog/_posts/2021-08-24-jq-select.md:377]: This is a error in the article I've been meaning to fix.

**Review by vladaionescu — APPROVED** [review]
This looks pretty good! I'm curious about how it performs in practice!

File: blog/_posts/2021-06-08-autoconf.md [hunk 2]
```diff
@@ -13,13 +13,14 @@ internal-links:
  - automake
  - m4
 topic: make
+last_modified_at: 2023-04-17
 ---
+**We're [Earthly](https://earthly.dev/). We make building software simpler and therefore faster using containerization. This article is about autotools and make. If you're interested in a different approach to build and packaging software then [check us out](/).**
```
Comment (jalletto) [inline blog/_posts/2021-06-08-autoconf.md:18]:
```suggestion
**We're [Earthly](https://earthly.dev/). We make building software simpler and therefore faster using containerization. This article is about autotools and make. If you're interested in a different approach to building and packaging software then [check us out](/).**
```

File: blog/_posts/2021-07-28-vscode-make.md [hunk 3]
```diff
@@ -9,9 +9,12 @@ internal-links:
 sidebar:
   nav: "makefile"
 topic: make
+last_modified_at: 2023-04-17
 ---
 <iframe width="560" height="315" src="https://www.youtube.com/embed/vAS4R5P0Orc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
 
+**We're [Earthly](https://earthly.dev/). We make building software simpler and therefore faster using containerization. This article is about using `make` and a `Makefile` in VS Code. If you're interested in a different approach to build and packaging software then [check us out](/).**
```
Comment (jalletto) [inline blog/_posts/2021-07-28-vscode-make.md:16]:
```suggestion
**We're [Earthly](https://earthly.dev/). We make building software simpler and therefore faster using containerization. This article is about using `make` and a `Makefile` in VS Code. If you're interested in a different approach to building and packaging software then [check us out](/).**
```

**Review by jalletto — APPROVED** [review]

**Comment by thtmnisamnstr** [comment]
This looks great. For the CTA text you added at the end "Next you might like to read more about a better way to do [docker multi-stage builds](https://643de9041830553751f54257--ephemeral-narwhal-3ac3b2.netlify.app/blog/docker-multistage) or if you’d like a faster way to build software, check out [Earthly](https://643de9041830553751f54257--ephemeral-narwhal-3ac3b2.netlify.app/):" You may want to add a section header over the top of it (Up Next), just to call it out. Up to you though. Maybe something you can test if we land on using text CTAs.

**Review by thtmnisamnstr — APPROVED** [review]

**Comment by adamgordonbell** [comment]
> You may want to add a section header over the top of it (Up Next), just to call it out.

Good idea. I'll keep it for next time. The python CSV article works like that actually.

**Pull request merged**